frontend/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js

```diff
@@ -138,12 +137,12 @@ export class NoticeOfWorkTable extends Component {
     {
       title: "NoW No.",
       dataIndex: "nowNum",
-      sortField: "trackingnumber",
+      sortField: "tracking_number",
       render: (text, record) => (
         <Link to={router.NOTICE_OF_WORK_INITIAL_APPLICATION.dynamicRoute(record.key)}>{text}</Link>
       ),
```

Comment:
Does this link still work? / does the component for the initial submission render correctly?

Reply:
yes, the link continues to work for the initial submission 👍